guides/doc-Release_notes/master.adoc

```diff
@@ -0,0 +1,30 @@
+include::common/attributes.adoc[]
+
+:numbered:
+:toc:
+:toc-placement: left
+
+ifndef::katello[]
```

Comment:
```suggestion
ifndef::katello[]
= Foreman {ProjectVersion} Release Notes

```

If you apply this suggestion, add `leveloffset=+1` when including foreman.adoc in the line below.

Reply:
It was intentional from my side to not have that indentation level. My reasoning is that you already have a page title which is `Foreman 3.0 Release Notes` So with the current version you end up with:
```text
Foreman 3.0 Release Notes
1. Headline Features
1.1. Puppet ENC functionality extracted to foreman_puppet plugin
```

What you're proposing is ending up with
```
Foreman 3.0 Release Notes
1. Foreman Release Notes
1.1. Headline Features
1.1.1. Puppet ENC functionality extracted to foreman_puppet plugin
```

Is that desired? Looks worse to me.

In the Katello case I chose to include it since there you have:
```
Foreman 3.0 and Katello 4.2 Release Notes
1. Foreman 3.0 Release Notes
1.1. Headline Features
1.1.1. Puppet ENC functionality extracted to foreman_puppet plugin
2. Katello 4.2 Release Notes
2.1. Headline Features
```

Reply:
OK, fine by me then.